lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -47,7 +51,8 @@ def start
 
   def client
     @_es ||= begin
-      adapter_conf = lambda {|f| f.adapter :excon }
+      excon_options = { client_key: @client_key, client_cert: @client_cert, client_key_pass: @client_key_pass }
+      adapter_conf = lambda {|f| f.adapter :excon, excon_options }
```

Comment:
True, I could specify the client_key and client_cert in the Faraday SSL hash.  However it appears Faraday doesn't have a parameter for a client_key_password so people would need to use an unencrypted client key.